content/en/docs/reference/access-authn-authz/rbac.md

```diff
@@ -14,34 +14,29 @@ Role-based access control (RBAC) is a method of regulating access to computer or
 
 {{% capture body %}}
 
-`RBAC` uses the `rbac.authorization.k8s.io` {{< glossary_tooltip text="API group" term_id="api-group" >}}
-to drive authorization decisions, allowing you to dynamically configure policies
-through the Kubernetes API.
+RBAC authorization uses the `rbac.authorization.k8s.io`
+{{< glossary_tooltip text="API group" term_id="api-group" >}} to drive authorization
+decisions, allowing you to dynamically configure policies through the Kubernetes API.
 
 To enable RBAC, start the {{< glossary_tooltip text="API server" term_id="kube-apiserver" >}}
 with `--authorization-mode=RBAC` (this is the default).
 
-## API Overview
+## API objects
 
-The RBAC API declares four top-level types which will be covered in this
-section. Users can interact with these resources as they would with any other
```

Comment:
was there a reason for dropping the "Users can interact..." text, and the mention of the bootstrapping section? That was added in response to user confusion that these are just normal API objects, and confusion around getting denied when attempting to play with the examples.

Reply:
Noted, thanks for explaining. I'll hyperlink to the section about bootstrapping.